1_fetch/src/fetch_dv_sites.R

```diff
@@ -16,8 +16,8 @@ fetch_dv_sites <- function(ind_file, dates){
         endDate = dates$end,
         parameterCd = "00060",
         statCd = "00003") %>%
-      dplyr::distinct() %>%
       dplyr::pull(site_no) %>%
+      unique() %>%
       c(sites)
```

Comment:
Is it possible that some sites show up in multiple HUCs? I don't know how that would happen, but since you're still seeing duplicates after many other fixes...